shared/getDeviceOnDash/hummingboard.md

```diff
@@ -1,13 +1,11 @@
-## Setting Up Your Device
-
 Put the SD card into your device, and connect either the ethernet cable or WiFi adapter. Now power up the device by inserting the micro usb cable.
```

Comment:
I'm obviously not familiar with how the substitutions / templating are being used in the docs, but should `SD card` here be `{{ $device.bootMedia }}` ? (same question applies to some of the other guides too)